### PR TITLE
feat: editable assets

### PR DESCRIPTION
**Pull request by richardtreier**
related to https://github.com/sovity/edc-extensions/issues/606

```[tasklist]
### Checklist
- [x] The PR title is short and expressive.
- [x] I have updated the CHANGELOG.md. See [changelog_update.md](https://github.com/sovity/edc-extensions/tree/main/docs/dev/changelog_updates.md) for more information.
- [x] I have updated the Deployment Migration Notes Section in the CHANGELOG.md for any configuration / external API changes.
- [x] I have performed a **self-review**
```

### CODE REVIEW

File: extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/ui/pages/asset/AssetUpdater.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    private final AssetIdValidator assetIdValidator;
+    private final SelfDescriptionService selfDescriptionService;
+
+    /**
+     * Creates an {@link Asset} from a {@link UiAssetCreateRequest}.
+     *
+     * @param request {@link UiAssetCreateRequest}
+     * @return {@link Asset}
+     */
+    public Asset createAsset(UiAssetCreateRequest request) {
```
Comment (kulgg) [inline extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/ui/pages/asset/AssetUpdater.java:38]: The class is called `AssetUpdater` but contains a method that doesnt have an asset argument and instead creates one from `UiAssetCreateRequest`

File: extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/WrapperExtensionContextBuilder.java [hunk 2]
```diff
@@ -30,6 +30,7 @@
 import de.sovity.edc.ext.wrapper.api.ui.UiResourceImpl;
 import de.sovity.edc.ext.wrapper.api.ui.pages.asset.AssetApiService;
 import de.sovity.edc.ext.wrapper.api.ui.pages.asset.AssetIdValidator;
```
Comment (kulgg) [inline extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/WrapperExtensionContextBuilder.java:32]: Import order not lexicographically
```
Error: eckstyle] [ERROR] /home/runner/work/edc-extensions/edc-extensions/extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/WrapperExtensionContextBuilder.java:33:1: Wrong lexicographical order for 'de.sovity.edc.ext.wrapper.api.ui.pages.asset.AssetBuilder' import. Should be before 'de.sovity.edc.ext.wrapper.api.ui.pages.asset.AssetIdValidator'. [CustomImportOrder]
```

**Review by kulgg — COMMENTED** [review]
```
Error: eckstyle] [ERROR] /home/runner/work/edc-extensions/edc-extensions/extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/WrapperExtensionContextBuilder.java:33:1: Wrong lexicographical order for 'de.sovity.edc.ext.wrapper.api.ui.pages.asset.AssetBuilder' import. Should be before 'de.sovity.edc.ext.wrapper.api.ui.pages.asset.AssetIdValidator'. [CustomImportOrder]

File: extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/WrapperExtensionContextBuilder.java (hunk 2 above)
Comment (richardtreier) [reply extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/WrapperExtensionContextBuilder.java]: should work now

**Pull request merged**